__test__/Notification.test.js

```diff
@@ -58,4 +61,125 @@ describe('Notification', () => {
             });
         });
     });
+
+    // interaction test
+    describe.only('interaction test', () => {
+        it('Can not find the close button by default', async () => {
+            await page.goto(
+                getPageURL(
+                    'toast',
+                    'positive',
+                    'single',
+                    '!true',
+                    '!false',
+                    '',
+                ),
+            );
+
+            // wait for element to be visible
+            await page.waitForSelector('daikin-notification', {
+                visible: true,
+            });
+
+            expect(
+                await page.$$(
+                    'daikin-notification >>> div[data-testid="daikin-notification"]',
+                ),
+            ).toHaveLength(1);
```

Comment:
I think this one is easier to understand.

```suggestion
            expect(
                await page.$(
                    'daikin-notification >>> div[data-testid="daikin-notification"]',
                )
            ).toBeTruthy();
```

Reply:
I had not thought of that method. Fixed.